packages/insomnia-inso/src/commands/generate-config.test.ts

```diff
@@ -190,6 +190,22 @@ describe('generateConfig()', () => {
       { type: 'declarative' }
     );
     expect(result).toBe(true);
+    expect(logger.__getLogs().log).toEqual([
+      `_format_version: \"1.1\"
+services: []
+upstreams: []
+`,
+    ]);
+  });
+
+  it('when format is json should generate declarative config as json', async () => {
```

Comment:
```suggestion
  it('should generate declarative config as json when format is set to json', async () => {
```

Generally the test names in this codebase follow a `it should foo` or `it does bar` pattern, given that's what the `it` keyword indicates